src/tests/Interop/COM/ComWrappers/WeakReference/WeakReferenceNative.cpp

```diff
@@ -167,8 +167,122 @@ namespace
             return UnknownImpl::DoRelease();
         }
     };
+
+    struct WeakReferenceSource : public IWeakReferenceSource, public IInspectable
```

Comment:
Do we need `IInspectable`?

Reply:
Yes we do. We pass this instance to the `WeakReference` struct above, which needs to take an `IInspectable*`.